cmd/kubemark/hollow-node.go

```diff
@@ -64,6 +64,7 @@ type hollowNodeConfig struct {
 	ProxierSyncPeriod    time.Duration
 	ProxierMinSyncPeriod time.Duration
 	NodeLabels           map[string]string
+	RegisterWithTaints   map[string]string
```

Comment:
You also need to propagate this to the kubelet flags, exactly like the NodeLabels is done - https://github.com/kubernetes/kubernetes/blob/8d54a84507c622e1552373d5d125c84ee22d1465/pkg/kubemark/hollow_kubelet.go#L164

Reply:
@mm4tt resolved :)

Reply:
@mm4tt : I changed it to type []core.Taint
Is it as intended?